CautionaryAlertsListener.Tests/FixtureConstants.cs

```diff
@@ -0,0 +1,12 @@
+namespace CautionaryAlertsListener.Tests
+{
+    public static class FixtureConstants
```

Comment:
If this is used only as part of the E2E tests then can we move this to be in the E2E folder

Reply:
It is used by other tests too (use cases, API fixtures, etc)